### PR TITLE
[FIX] Adding more translations

### PR DESCRIPTION
**Pull request by ViktorHek**
adding more translations

### CODE REVIEW

File: public/locales/sv/translation.json [hunk 1]
```diff
@@ -6,5 +6,22 @@
...
+  "submit": "Logga in",
+  "sign_up": "Bli Medlem",
+  "create_account": "Skapa ett Konto",
+  "submit_reg": "Registrera Dig",
+  "login_here": "Logga In",
+  "subscribe_sub": "Prenumerera",
+  "pay_header": "Ge oss garn och bli en medlem",
+  "750_sek_year": "För bara 750 kr/år får du full tillgång till Kitty News!",
+  "card_number": "Kortnummer",
+  "expiry_date": "Utgångs Datum",
```
Comment (Ehugo2000) [inline public/locales/sv/translation.json]: Utgångsdatum

File: public/locales/sv/translation.json [hunk 2]
```diff
@@ -6,5 +6,22 @@
...
+  "cvc_code": "CVC Kod",
+  "confirm_payment": "Bekräfta Betalning",
+  "email": "E-post",
+  "password": "Lösenord",
+  "password_confirmation": "Lösenords Bekräftelse"
```
Comment (Ehugo2000) [inline public/locales/sv/translation.json]: Bekräfta lösenord

File: public/locales/en/translation.json [hunk 3]
```diff
@@ -6,5 +6,22 @@
   "culture": "Culture",
   "empty_index_message": "Sorry, there's nothing to see here yet!",
   "footer": "Made with React",
```
Comment (Ehugo2000) [inline public/locales/en/translation.json]: Adjust content to our site?

File: src/modules/Authentication.js [hunk 4]
```diff
@@ -38,7 +38,7 @@ const logIn = async (event) => {
       payload: response,
     })
     store.dispatch({ type: 'CLOSE_LOGIN_FORM' })
-    alert('Welcome, cute kitty!')
+    alert( 'Welcome, cute kitty!')
```
Comment (Ehugo2000) [inline src/modules/Authentication.js]: Looks like it is a space here?!

**Review by Ehugo2000 — COMMENTED** [review]
Nice work! just some spelling thingy!

File: src/modules/Authentication.js (hunk 4 above)
Comment (ViktorHek) [reply src/modules/Authentication.js]: done :)

File: public/locales/en/translation.json (hunk 3 above)
Comment (ViktorHek) [reply public/locales/en/translation.json]: done :)

File: public/locales/sv/translation.json (hunk 2 above)
Comment (ViktorHek) [reply public/locales/sv/translation.json]: done :)

File: public/locales/sv/translation.json (hunk 1 above)
Comment (ViktorHek) [reply public/locales/sv/translation.json]: donr :)

**Pull request merged**